packages/cli/src/commands/project/list.ts

```diff
@@ -48,10 +51,38 @@ export default async function list(
 
   const elapsed = ms(Date.now() - start);
 
+  if (deprecated) {
+    const upcomingDeprecationVersions = NODE_VERSIONS.filter(
+      nodeVersion =>
+        nodeVersion.discontinueDate &&
+        nodeVersion.discontinueDate.valueOf() > Date.now()
+    );
+    const upcomingDeprecationVersionsList = upcomingDeprecationVersions.map(
+      nodeVersion => nodeVersion.range
+    );
+    console.log(projectList);
+    projectList = projectList.filter(
+      project =>
+        project.nodeVersion &&
+        upcomingDeprecationVersionsList.includes(project.nodeVersion)
+    );
+
+    output.log(
+      `The following Node.js versions will be deprecated soon: ${upcomingDeprecationVersionsList.join(
```

Comment:
I feel this needs to be called out better. Either an `!` or colored yellow or something.